### PR TITLE
fix(5568): Sistent buttons spacing

### LINKED ISSUE #5568: [Sistent]: Buttons example don't have gap in between
### Description

Buttons example don't have gap in between
### Expected Behavior

Their should be gap in between the button (gap size needs to be finalize)
image

### Screenshots

image

### Environment:
- Host OS: any
- Browser: any

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by surajgjadhav**
**Description**

This PR fixes #5568

**Notes for Reviewers**
Before Fix:
image

After Fix:
image

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 51217ae062f44f8b361aa8515b92fb6806a40a39 at: https://66699dd19992cc23e224e2ad--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 692002f6d25332778425b93d82be0a196f9b8e44 at: https://666ac6d5ac27ded85f11dd8c--layer5.netlify.app

**Comment by Deepak0320Singhal** [comment]
@surajgjadhav Thank you for your contribution.
Let's discuss this on Websites's call on Monday. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) .

File: src/sections/Projects/Sistent/about.js [hunk 1]
```diff
@@ -42,27 +40,47 @@ const SistentAbout = () => {
...
+              Sistent leverages Material UI libraries and provides a custom
+              theme on top of it for a consistent look and feel. It includes
+              components, icons, and design tokens that developers can readily
+              integrate into their applications. By using Sistent, developers
+              can save time and effort while maintaining a high-quality user
+              experience throughout Layer5 products.
             </p>
             <p>
-              <b>Sistent provides a consistent user experience across Layer5 projects and any frontend software projects that choose to use Sistent, too.</b>
+              <b>
```
Comment (sudhanshutech) [inline src/sections/Projects/Sistent/about.js:58]: please try to avoid do lint changes, it make difficult to review

**Review by sudhanshutech — APPROVED** [review]

File: src/sections/Projects/Sistent/about.js (hunk 1 above)
Comment (surajgjadhav) [reply src/sections/Projects/Sistent/about.js:58]: Sure, will take  care of that

**Pull request merged**